src/components/TrackTile/TrackerDetails/TrackAmountControl.tsx

```diff
@@ -24,7 +31,13 @@ const TrackAmountControl: FC<Props> = ({ color, value, onChange }) => {
 
   const [currentValue, setCurrentValue] = useState(numberFormat(value));
 
-  useEffect(() => setCurrentValue(numberFormat(value)), [value]);
+  useEffect(() => {
+    if (!saveInProgress) {
+      debounce(() => {
```

Comment:
I did away with the debounce in my next PR because it caused issues with the initial syncing.